tests/protocols/test_websocket.py

```
@@ -529,6 +529,52 @@ async def websocket_session(url: str):
+    """
+    send_error: BaseException | None = None
+
+    async def app(scope: Scope, receive: ASGIReceiveCallable, send: ASGISendCallable):
+        nonlocal send_error
+        message = await receive()
+        assert message["type"] == "websocket.connect"
+        # Give the client time to drop the connection before we accept it.
+        await asyncio.sleep(0.1)
+        try:
+            await send({"type": "websocket.accept"})
+        except BaseException as exc:  # pragma: no cover - recorded for the assertion
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)